controllers/ingressnodefirewall_controller.go

```diff
@@ -103,6 +103,7 @@ func (r *IngressNodeFirewallReconciler) Reconcile(ctx context.Context, req ctrl.
 			continue
 		}
 
+		nodeState := ingressNodeFirewallCurrentNodeState.DeepCopy()
 		// If the node name was found ...
 		// a) compare the specs or owner reference. If the specs or owner reference are different, update the current spec.
 		if !equality.Semantic.DeepEqual(ingressNodeFirewallCurrentNodeState.Spec, desiredNodeState.Spec) ||
```

Comment:
I know it doesn't change anything to the correctness of the code, but let's make it "nodeState" everywhere then for consistency and readability even in the `if` clauses?
In line 109 right now we use `ingressNodeFirewallCurrentNodeState.Spec` for the comparison but then in line 135 `!equality.Semantic.DeepEqual(nodeState.Status, desiredNodeState.Status) `